### DISCUSSION TITLE
GitHub Pages Root

### DISCUSSION THREAD

**Discussion by codeadamca** [community/community#63631] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When I use GitHb Pages without a custom domain name, the content becomes available at `https://<USERNAME>.github.io/<REPO_NAME>/`. Is there anyway to post content at `https://<USERNAME>.github.io/`?

Same question applies to organizations. Could I post content at `https://<ORGANIZATION>.github.io/`?

When I go to either of these addresses I get a 404 page:

<img width="646" alt="404" src="https://github.com/community/community/assets/12630767/1270e34e-f538-4952-a687-ad922d5c9198">

It would be nice if I could. Then I could add a `favicon.ico` file or custom 404 page that would apply to all my GitHub Pages sites.

**Comment by MayaAtwan** [comment, marked as answer]
I found this solution and I hope it works,
Instructions for setting up username.github.com *
  Create a repo named username.github.com
  Push a `master` branch to GitHub and enjoy!
Instructions for setting up username.github.com/repo-name *
Caution: make your working directory clean before you do this (either stash or commit), otherwise this will lose any changes you've made to your project since the last commit.
  cd /path/to/repo-name
  git symbolic-ref HEAD refs/heads/gh-pages
  rm .git/index
  git clean -fdx
  echo "My GitHub Page" > index.html
  git add .
  git commit -a -m "First pages commit"
  git push origin gh-pages

https://gist.github.com/atomize/2012391

**Reply by codeadamca** [reply]
Awesome thanks! Just in case anyone is looking in the future... 

This works as described for both `https://<USERNAME>.github.io/` and `https://<ORGANIZATION>.github.io/`. Notice that the domain is `.io` not `.com`. The repo needs to be named `<USERNAME>.github.io` and `<ORGANIZATION>.github.io`. 

This also worked as I was hoping for a `favicon.ico` file. If you put a `favicon.ico` in your `<USERNAME>.github.io` repo, it will be used for all repos using GitHub Pages without a custom domain name. 

I added full details on a Gist:
https://gist.github.com/codeadamca/bc3dcb6fc768ee1714918aa15cf13fad
